examples/file-and-s3-upload/app/routes/s3-upload.tsx

```diff
@@ -0,0 +1,59 @@
+import type { ActionFunction, UploadHandler } from "@remix-run/node";
+import {
+    json,
+  unstable_composeUploadHandlers as composeUploadHandlers,
+  unstable_createMemoryUploadHandler as createMemoryUploadHandler,
+  unstable_parseMultipartFormData as parseMultipartFormData,
+} from "@remix-run/node";
+import { Form, useActionData } from "@remix-run/react";
+import { s3UploadHandler } from "~/utils/s3.server";
+
+type ActionData = {
+  errorMsg?: string;
+  imgSrc?: string;
+  imgDesc?: string;
+};
+
+export const action: ActionFunction = async ({ request }) => {
+  const uploadHandler: UploadHandler = composeUploadHandlers(
+    s3UploadHandler,
+    createMemoryUploadHandler()
+  );
+  const formData = await parseMultipartFormData(request, uploadHandler);
+  const imgSrc = formData.get("img");
+  const imgDesc = formData.get("desc");
+  console.log(imgDesc)
+  if (!imgSrc) {
+    return json({
+      error: "Something went wrong while uploading",
+    });
+  }
+  return json({
+    imgSrc,
+    imgDesc,
+  });
+};
+
+export default function Index() {
+  const data = useActionData<ActionData>();
```

Comment:
Because this is not a navigation we should use `useFetcher` instead. Could you update this example to use that instead?

Reply:
Hey Kent, thanks for reviewing this! I've updated the example to use `useFetcher` instead